save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -26,4 +27,5 @@ data class GeneralConfig(
     val suiteName: String,
     val excludedTests: String = "",
     val includedTests: String = "",
+    val ignoreSaveComments: Boolean = false,
```

Comment:
@kgevorkyan after all, are we making these fields nullable? Or you will handle it in #79 ?

Reply:
Yes, we decide to make such options nullable for now. I can handle it, if this PR will be merged first. Or @Cheshiriks can change it by his own